aws_guardduty/lib/botocore/data/importexport/2010-06-01/paginators-1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "pagination": {
+    "ListJobs": {
+      "input_token": "Marker",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 635d1d20-630e-41da-9aac-ddbaafc1215a -->

Reply:
This is an example Token part of the AWS SDK library.